### PR TITLE
Removed MathJax js console error issue

### PR DESCRIPTION
**Pull request by ubaidsworld**
Mathjax error were visible in the console and now the error is gone.

## Description

Mathjax error were visible in the console and now the error is gone.

## Motivation and Context

There were many issues regarding the js library used in the visdom and it was showing console error. I do not like the console errors. Hence, thought of removing it.
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] For JavaScript changes, I have re-generated the minified JavaScript code.

### CODE REVIEW

File: py/visdom/server.py [hunk 1]
```diff
@@ -1743,6 +1743,12 @@ def download_scripts(proxies=None, install_dir=None):
                 logging.error('Error {} while downloading {}'.format(
                     exc.reason, key))
 
+    filename = '%s/static/%s/%s' % (install_dir, "js", "MathJax.js")
+    if not os.path.exists(filename):
+        mathjax_js = requests.get("https://raw.githubusercontent.com/mathjax/MathJax/master/es5/tex-mml-svg.js")#("https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_SVG")
```
Comment (JackUrb) [inline py/visdom/server.py]: Is it not sufficient to just replace the url in line 1656 with this one?

**Comment by JackUrb** [comment]
Thanks for taking another stab at this - almost ready to accept it out right, just one question of curiosity on the implementation. 

If my suggestion doesn't work, just remove the commented lines 1656-1657 entirely and then remove the additional comment on 1748. Also, you should use a more descriptive variable name than `filename` considering it's specifically the mathjax file, such as `mathjax_filename`.

File: py/visdom/static/index.html [hunk 2]
```diff
@@ -29,12 +29,12 @@
...
 
-    <!-- Mathjax -->
-    <script type="text/javascript" async src={{ static_url("js/mathjax-MathJax.js") }}></script>
-
     <!-- Plotly -->
     <script src={{ static_url("js/plotly-plotly.min.js") }}></script>
 
+    <!-- Mathjax -->
+    <script type="text/javascript" async src={{ static_url("js/MathJax.js") }}></script>
+
```
Comment (JackUrb) [inline py/visdom/static/index.html]: It's important that the `MathJax` library is imported _before_ the `Plotly` library, so please don't swap the ordering here.

Comment (ubaidsworld) [reply py/visdom/static/index.html]: @JackUrb  I do not ant to swap the lines. But what I found I will share with you.
Actually, while I was testing the edited code for mathjax, I have found that Plotly suddenly gave some error and that were logged in the console.
But when I swapped the loading positions of the JS, it was working fine.
But if you think that it is not appropriate then I will make the changes. 
Here take a look at this error in the console when plotly is placed after the Mathjax:
image

Let me know and I will make the changes if you say so.

File: py/visdom/server.py (hunk 1 above)
Comment (ubaidsworld) [reply py/visdom/server.py]: I guess it works.

File: py/visdom/static/index.html (hunk 2 above)
Comment (JackUrb) [reply py/visdom/static/index.html]: Ah okay so there's still something weird going on with this import - The entire goal is to be able to use `LaTeX` in Plotly, and `MathJax` is supposed to enable that. When `MathJax` is imported before Plotly, Plotly attempts to integrate with the library.

File: py/visdom/server.py (hunk 1 above)
Comment (ubaidsworld) [reply py/visdom/server.py]: Just give me a little time I will fix this.

File: py/visdom/static/index.html (hunk 2 above)
Comment (ubaidsworld) [reply py/visdom/static/index.html]: ok .. I got it ... just give me some time and it will be fixed.. Insha allah..

**Comment by JackUrb** [comment]
Alright digging into this a little more, I suspect the attempt to move MathJax to a local download in order to help those operating behind firewalls was somewhat incomplete. `MathJax.js` relies on the directory structure hosted on a cdn like `https://cdn.jsdelivr.net/npm/mathjax@2.7.1/` to load the correct resources, thus the fix in 2d12134 broke `MathJax`. If we wanted to get this to work locally, we'd need to download the entire package to a local directory.

For now, I think it'd be sufficient to go back to importing directly from a CDN in py/visdom/static/index.html. `https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_SVG` or `https://cdn.jsdelivr.net/npm/mathjax@2.7.1/MathJax.js?config=TeX-AMS-MML_SVG` would both be sufficient.

**Comment by ubaidsworld** [comment]
@JackUrb Yes, I got it . I will add the MathJax folder separately in the project and that will do the magic. I am closing this request and will pull a new request, with the proper addons.

**Pull request closed without merging**